projects/kit/pipes/iso-to-country-code/iso-to-country-code.pipe.ts

```diff
@@ -1,18 +1,18 @@
 import {Inject, Pipe, PipeTransform} from '@angular/core';
 import {TuiCountryIsoCode} from '@taiga-ui/i18n';
-import {MASK_AFTER_CODE_REGEXP} from '@taiga-ui/kit/constants';
 import {TUI_COUNTRIES_MASKS} from '@taiga-ui/kit/tokens';
+import {tuiIsoToCountryCode} from '@taiga-ui/kit/utils';
 
 @Pipe({
     name: `tuiIsoToCountryCode`,
 })
 export class TuiIsoToCountryCodePipe implements PipeTransform {
     constructor(
         @Inject(TUI_COUNTRIES_MASKS)
-        readonly countriesMasks: Record<TuiCountryIsoCode, string>,
+        private readonly countriesMasks: Record<TuiCountryIsoCode, string>,
     ) {}
 
     transform(isoCode: TuiCountryIsoCode): string {
-        return this.countriesMasks[isoCode].replace(MASK_AFTER_CODE_REGEXP, ``);
+        return tuiIsoToCountryCode(this.countriesMasks, isoCode);
```

Comment:
why not from options `this.options.isoToCountryCode`

Reply:
pipe does not have options